### ISSUE TITLE
[feature] Automatic Subscriptions

### ISSUE THREAD

**Issue by GeckoBoy84** [Ctrlpanel-gg/panel#138]
# 🚀 Feature Request

### Is your proposal related to a problem?

Would be nice to have more knowledge on likely income for the month.

### Describe the solution you'd like

Options to enable automatic subscriptions for PayPal with an option to add extra credits for using a subscription with the amount configurable. Would also be nice to have additional rewards for having auto subscriptions setup for x months

**Comment by William071802** [comment]
This would be incredible, not sure how it would work with regards to editing the store during active subscriptions but it seems worth discussing. Could perhaps have a sunset system where each sub lasts x months (with options) and you can hide options from the store but not delete them until everyone's subscription to it has ended

**Comment by RetlyCompany** [comment]
Yes, this would be great since I've noticed I'm losing clients solely because they must access to the dashboard to buy credits. I've already discussed with one of the developers in the hosting team about adding this option ourselves, however It would be really nice to have it natively.

**Comment by 1day2die** [comment]
im not 100% sure about this, IceToast can elaborate further on this, but i think with the Stripe integration this might be a thing

**Comment by S0ly** [comment]
#906 was a duplicate but whit more details

**Comment by sybetjeh** [comment]
whole point of Ctrlpanel.gg is that it's credit based and not billed directly. This is not what Ctrlpanel is

If you're looking for a system like this go to a paid solution like Blesta or WHMCS.

**Comment by simbabimba-dev** [comment]
![bruh](https://cdn.discordapp.com/emojis/1294797296332177500.webp?size=64)

**Comment by MrWeez** [comment]
I'm not sure if it's that easy to implement, given that not every payment gateway supports regular payments. I am more inclined to the option that this will not be added due to the difficulties of implementing the functionality. Needs to be discussed and after that status can be updated
